packages/components/src/html/index.html

```diff
@@ -9,29 +9,1427 @@
     <title>Stencil Component Starter</title>
```

Comment:
This is probably a mistake. Please let's leave this `index.html` file as it is in `main` 🤓

Reply:
That wasn't meant to be... I restored the original index.html